.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
```

Comment:
Nice work @sa-ma. I can see you are using babel 7. Won't we be needing @babel/transform-runtime and @babel/runtime? To avoid regenerator runtime error when transpiling to async functions. since @eokwukwe said we'll be working with async await.

Reply:
@sa-ma. You need to implement @uthdev feedback. Do that so that I can merge this `PR`

Reply:
Fixed @uthdev  I added a target version for the babel-presets so it will generate all the plugins needed. There won't be any need to install the plugins individually